content/en/release-notes/log4j-patches.md

```diff
@@ -0,0 +1,177 @@
+---
+date: '2021-04-24T00:00:00Z'
+section_menu: releases
+menu:
+  projects:
+    name: Releases
+    weight: 9
+    identifier: logj4patches
+    project: releases
+    project: releases
+version: 'releases'
+title: Log4j patches
+---
```

Comment:
I think the 404 _is_ front matter related, and I think it might be indentation related.

Reply:
I think the second 'project: releases' heading should not be indented. I've attached screenshots in Teams from other pages.